### DISCUSSION TITLE
No output in the Serie Monitor P04-Arduino

### DISCUSSION THREAD

**Discussion by HelenaRD33** [TheAlbertDev/MASB#50] [Q&A] [answered]
Good afternoon Albert,

When executing the code for the serial communication in the Arduino part of the lab 4 I don't get any values as ouputs in the Serie Monitor when turning the potentiometer knob.
I have made sure that the code was as in the README document and I think other groups have the same problem.

Thank you in advance,

Helena

**Comment by Laiaborrell** [comment]
I have the same problem, could not fix it either

**Comment by Laiaborrell** [comment, marked as answer]
Hi Helena, I have changed this value on the image to 9600 and now the terminal shows the outputs. 
![image](https://user-images.githubusercontent.com/78802600/110234370-62ed3980-7f2a-11eb-9d27-f0037835daff.png)

**Comment by HelenaRD33** [comment]
Thanks Laia!!
